src/overlays/actors/ovl_En_Insect/z_en_insect.h

```diff
@@ -9,6 +9,16 @@ struct EnInsect;
 
 typedef void (*EnInsectActionFunc)(struct EnInsect*, GlobalContext*);
 
+#define INSECT_FLAG_0 (1 << 0)
+#define INSECT_FLAG_1 (1 << 1)
+#define INSECT_FLAG_IS_SHORT_LIVED (1 << 2)
```

Comment:
in general Id be fine with any mention of `insect` being `bug`, so these enums and the instance variables

Reply:
tbh I'm not a fan of using "bug", solely because of the other meanings in computer science

Reply:
i agree its a little weird, but I think context will be enough. no one calls these insect, its always bugs